src/components/ChatSidebar/ChatSidebarLink.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+import { Star } from "lucide-react";
+import { SidebarMenuButton, SidebarMenuItem } from "@/components/ui/sidebar";
+
+export function ChatSidebarLink({
+  href,
+  title,
+}: Readonly<{ href: string; title: string }>) {
+  return (
+    <SidebarMenuItem>
+      <SidebarMenuButton asChild>
+        <Link href={href}>
```

Comment:
I'm not sure if we will need link there, I think that a button itself will be enough (to rerender chat) - but for now it's fine

Reply:
how would that work though?

Reply:
By passing function to button `onClick` method which will update the state in chat component, I guess.
Update function would take data from local storage and cause chat rerender.
But maybe I'm wrong, it would be good to hear @maciejkrol18

Reply:
I will wait for his review before I merge this

Reply:
I think it that how we handle this depends on how we're going to store user chats. If i recall correctly, currently we're going for a local storage approach, with all the chats being stored in user's browser, as to not have a need for implementing any auth and database. If we're going to store them in an array of objects, where each object represents a conversation with an UUID and rest of any relevant data (such as the messages), we could go for an approach resembling ChatGPT.

In ChatGPT, the users starts at an initial view with just the prompt bar ("Chat" view equivalent in our Figma design) with the route being `chatgpt.com`. Once the user submits their prompt, new conversation with an UUID gets created and the user gets redirect to a new route being `chatgpt.com/c/UUID` ("Chat with promochator" view equivalent   in our Figma design), which shows the conversation. 

How would this apply to our project? The "root" route would be `promochator.solvro.pl/chat` (equivalent of `chatgpt.com`). Each link in the sidebar would point to `promochator.solvro.pl/chat/UUID`. Upon entering this route, we would check if a conversation with matching UUID exists in user's browser storage and render it if found. "New chat" button would just point to `promochator.solvro.pl/chat`. 

If we would go for such approach then this piece of code is fine.